src/verifier/compare_outputs.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+if [ $# -eq 0 ]; then
+    echo "Missing argument for name of existing CSV file from verified_output to check against."
+    exit $1
```

Comment:
Good catch on the `$1`. Full documentation of usage is in the `README` docs. Sure, could duplicate that here too.